CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#4487](https://github.com/CocoaPods/CocoaPods/pull/4487)
 
+* Add support for specifying :source with a pod dependency.  
```

Comment:
I think your changelog entry should be at the top in order to preserve historical flow.